admin_guide/managing_networking.adoc

```diff
@@ -172,10 +183,10 @@ name of `router` for the deployment configuration and the service-account):
 $ oc env dc router ROUTER_ENABLE_INGRESS=true`
 ----
 
-. Add the cluster-admin role to the router:
+. Add the cluster-admin role to the router, where `-z` is the service account:
 +
 ----
-$ oc adm policy add-role-to-user cluster-admin router`
+$ oc adm policy add-cluster-role-to-user cluster-reader -z router
```

Comment:
`oc adm policy add-cluster-role-to-user system:openshift:controller:service-serving-cert-controller -z router` is also needed for enabling ingress, otherwise the router pod logs will show many errors below:
```
E0323 06:22:50.408710       1 reflector.go:205] github.com/openshift/origin/pkg/router/controller/factory/factory.go:117: Failed to list *core.Secret: secrets is forbidden: User "system:serviceaccount:default:router" cannot list secrets at the cluster scope: User "system:serviceaccount:default:router" cannot list all secrets in the cluster
E0323 06:22:51.410501       1 reflector.go:205] github.com/openshift/origin/pkg/router/controller/factory/factory.go:117: Failed to list *core.Secret: secrets is forbidden: User "system:serviceaccount:default:router" cannot list secrets at the cluster scope: User "system:serviceaccount:default:router" cannot list all secrets in the cluster
E0323 06:22:52.417695       1 reflector.go:205] github.com/openshift/origin/pkg/router/controller/factory/factory.go:117: Failed to list *core.Secret: secrets is forbidden: User "system:serviceaccount:default:router" cannot list secrets at the cluster scope: User "system:serviceaccount:default:router" cannot list all secrets in the cluster
```

others looks good.